spec/features/approving_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+RSpec.feature 'Approving' do
+  let(:admin) do
```

Comment:
I am in favor of disabling the rubocop rule that requires `do ... end` on multi line `let` blocks - in specs only.

Reply:
Done.